client/src/store/index.js

```diff
@@ -1,16 +1,15 @@
-import { createStore, applyMiddleware, compose } from 'redux';
+import { createStore, applyMiddleware } from 'redux';
 import createSagaMiddleware from 'redux-saga';
 import rootReducer from './reducers/index';
 import rootSaga from './sagas/index';
+import { composeWithDevTools } from 'redux-devtools-extension';
 
 const sagaMiddleware = createSagaMiddleware();
 
-const reduxDevTools = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
-
 const store = createStore(
   rootReducer,
-  compose(applyMiddleware(sagaMiddleware), reduxDevTools)
-);
+  composeWithDevTools(applyMiddleware(sagaMiddleware))
+  );
```

Comment:
No need to assign DevTools as a constant and include it in the compose, composeWithDevTools alone is sufficient